.github/workflows/publish.yml

```diff
@@ -24,8 +24,8 @@ jobs:
       - name: Publish Extension as Patch
         run: yarn vsce:check "latest"
 
-  publish-dev:
-    name: Publish Dev
+  publish-insider:
```

Comment:
intentional in this PR?

(also more locations further down in the diff)

Reply:
no, not intentional